components/esp_modem/examples/modem_console/sdkconfig.defaults

```diff
@@ -1,8 +1,12 @@
-# Override some defaults to enable PPP
+# This file was generated using idf.py save-defconfig. It can be edited manually.
+# Espressif IoT Development Framework (ESP-IDF) Project Minimal Configuration
+#
+CONFIG_IDF_TARGET="esp32c3"
+CONFIG_EXAMPLE_MODEM_DEVICE_SIM7600=y
+CONFIG_EXAMPLE_MODEM_UART_TX_PIN=4
+CONFIG_EXAMPLE_MODEM_UART_RX_PIN=5
+CONFIG_EXAMPLE_MODEM_UART_EVENT_TASK_STACK_SIZE=4096
+CONFIG_ESP_MAIN_TASK_STACK_SIZE=7168
```

Comment:
Please delete all fields that are overriding defaults from Kconfig. And the IDF_TARGET too.